### PR TITLE
[Tests-Only] Fix test scenarios with same title on same file

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Fix test scenarios with same title on same file

## Related Issue

- Fixes scenarios with the same name

## Motivation and Context

this will help an automated tool to update expected failure files

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13929/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13929/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13934/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13934/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13940/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13953/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13963/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14011/

<p>





</p></details>

File: tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature [hunk 1]
```diff
@@ -136,8 +136,8 @@ Feature: Resharing shared files with different permissions
       | shared-entry-name | received-entry-name |
       | simple-folder     | simple-folder (2)   |
       | lorem.txt         | lorem (2).txt       |
-      | simple-folder     | simple-folder (2)   |
-      | lorem.txt         | lorem (2).txt       |
+
+
```
Comment (Talank) [inline tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature:142]: the gap is big. But not keeping this gap will create a lot of changes in the expected failure file.

**Review by individual-it — APPROVED** [review]

File: tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature (hunk 1 above)
Comment (dpakach) [reply tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature:142]: Yes, exactly, I'm trying not to change any line numbers right now.

**Pull request merged**